docs/user-guide/multiple_sources.md

```diff
@@ -0,0 +1,62 @@
+# Multiple Sources for an Application
```

Comment:
Should not we update the docs/operator-manual/application.yaml? or maybe it's auto-generated?